tests/src/deviceLib/hipDoublePrecisionIntrinsics.cpp

```diff
@@ -34,34 +34,38 @@ THE SOFTWARE.
 #pragma clang diagnostic ignored "-Wunused-variable"
 
 __device__ void double_precision_intrinsics() {
-    // __dadd_rd(0.0, 1.0);
-    // __dadd_rn(0.0, 1.0);
-    // __dadd_ru(0.0, 1.0);
-    // __dadd_rz(0.0, 1.0);
-    // __ddiv_rd(0.0, 1.0);
-    // __ddiv_rn(0.0, 1.0);
-    // __ddiv_ru(0.0, 1.0);
-    // __ddiv_rz(0.0, 1.0);
-    // __dmul_rd(1.0, 2.0);
-    // __dmul_rn(1.0, 2.0);
-    // __dmul_ru(1.0, 2.0);
-    // __dmul_rz(1.0, 2.0);
+#if defined OCML_BASIC_ROUNDED_OPERATIONS
+    __dadd_rd(0.0, 1.0);
+    __dadd_rn(0.0, 1.0);
+    __dadd_ru(0.0, 1.0);
+    __dadd_rz(0.0, 1.0);
+    __ddiv_rd(0.0, 1.0);
+    __ddiv_rn(0.0, 1.0);
+    __ddiv_ru(0.0, 1.0);
+    __ddiv_rz(0.0, 1.0);
+    __dmul_rd(1.0, 2.0);
+    __dmul_rn(1.0, 2.0);
+    __dmul_ru(1.0, 2.0);
+    __dmul_rz(1.0, 2.0);
+#endif
     __drcp_rd(2.0);
```

Comment:
Why is drcp not hidden?  I don't think there is a correct implementation

Reply:
Hi Brian, where can I find the ocml functions for rcp?

Reply:
There isn't one.  A standard reciprocal is simply 1 / x.  There is no way to get a non-default-rounded reciprocal at all (just like there is no way to get a non-default-rounded add).

Reply:
I.e., when OCML implements non-default-rounded division functions, then these reciprocal functions should be implemented using it.